profiler/hotapp/main.go

```diff
@@ -133,7 +145,10 @@ func main() {
 	}
 
 	// Simulate some memory allocation.
-	alloc()
+	allocOnce()
+
+	// Simulate repeated memory allocations.
+	go allocMany()
```

Comment:
I like how this leaves concurrency up to the caller of the function, rather than creating the goroutine inside of `allocMany`. :+1: